### PR TITLE
apt: add Dutch translation

### PR DESCRIPTION
**Pull request by leonvsc**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

File: pages.nl/linux/apt.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+> Voor gelijkwaardige commando's in andere pakket managers, zie <https://wiki.archlinux.org/title/Pacman/Rosetta>.
+> Meer informatie: <https://manpages.debian.org/latest/apt/apt.8.html>.
+
+- Werk de lijst van beschikbare pakketten en versies bij (het wordt aanbevolen dit uit te voeren voor elk ander `apt` commando):
+
+`sudo apt update`
+
+- Zoek naar een bepaald pakket:
+
+`apt search {{package}}`
```
Comment (EmilySeville7cfg) [inline pages.nl/linux/apt.md:14]:
```suggestion
- Zoek naar een specifiek pakket:

`apt search {{pakket}}`
```

Optional fix (error source: original page): broken description: `specific` expected.
Note: translation source: Search for a specific package.

File: pages.nl/linux/apt.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+- Toon informatie over een pakket:
+
+`apt show {{package}}`
```
Comment (EmilySeville7cfg) [inline pages.nl/linux/apt.md:18]:
```suggestion
- Toon informatie voor een specifiek pakket:

`apt show {{pakket1 pakket2 ...}}`
```

Almost the same issue as above.
Mandatory fix (error source: original page): [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases): ellipsis expected.
Note: translation source: Show information for a specific package.

File: pages.nl/linux/apt.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+- Installeer of werk het pakket bij naar de laatste beschikbare versie:
+
+`sudo apt install {{package}}`
```
Comment (EmilySeville7cfg) [inline pages.nl/linux/apt.md:22]:
```suggestion
- Installeer specifieke pakketten of werk ze bij naar de nieuwste beschikbare versies:

`sudo apt install {{pakket1 pakket2 ...}}`
```

The same issue as above.
Note: translation source: Install specific packages, or update them to the latest available versions.

File: pages.nl/linux/apt.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- Een pakket verwijderen (door in plaats daarvan `purge` te gebruiken, worden ook de configuratiebestanden verwijderd):
+
+`sudo apt remove {{package}}`
```
Comment (EmilySeville7cfg) [inline pages.nl/linux/apt.md:26]:
```suggestion
- Verwijder specifieke pakketten (gebruik in plaats daarvan purge om ook hun configuratiebestanden te verwijderen):

`sudo apt remove {{pakket1 pakket2 ...}}`
```

The same issue as above.
Note: translation source: Remove specific packages (use `purge` instead to also remove their configuration files).

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! ❤️

File: pages.nl/linux/apt.md (hunk 1 above)
Comment (leonvsc) [reply pages.nl/linux/apt.md]: Thanks for the review! Fixed in the latest commit.

File: pages.nl/linux/apt.md (hunk 2 above)
Comment (leonvsc) [reply pages.nl/linux/apt.md]: Thanks for the review! Fixed in the latest commit.

File: pages.nl/linux/apt.md (hunk 3 above)
Comment (leonvsc) [reply pages.nl/linux/apt.md]: Thanks for the review! Fixed in the latest commit.

File: pages.nl/linux/apt.md (hunk 4 above)
Comment (leonvsc) [reply pages.nl/linux/apt.md]: Thanks for the review! Fixed in the latest commit.

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**